bundles/azure-service-broker-0.0.1/chart/azure-service-broker/templates/post-install-job.yaml

```diff
@@ -59,3 +59,58 @@ spec:
                         echo "ServiesBroker is not ready. Timeout reached";
                         exit 1;
                       fi
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+    name: azure-service-broker-docu-checker
+    labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: {{ .Release.Name }}
+        heritage: {{ .Release.Service }}
+    annotations:
+        "helm.sh/hook": post-install,post-upgrade
+        "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+    template:
+        metadata:
+            annotations:
+                sidecar.istio.io/inject: "false"
+        spec:
+            serviceAccountName: {{ include "fullname" . }}-docu-checker
+            restartPolicy: Never
+            containers:
+                - name: docu-checker
+                  image: eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20190325-ff66a3a
+                  imagePullPolicy: IfNotPresent
+                  command: ["/bin/sh","-c"]
+                  args:
+                      - |
+                          while true
+                          do
+                            echo "Get list of not ready DocsTopic:"
+                            lines=$(kubectl get DocsTopic -l chart=${LABEL_CHART} -l release=${LABEL_RELEASE} -n ${NS} \
```

Comment:
if you specify the `-l` flag multiple times then the last one will be taken into account.

If you want to have AND statement you need to write `-l chart=${LABEL_CHART},release=${LABEL_RELEASE}`